remote_challenge_evaluation/evaluate.py

```diff
@@ -1,4 +1,3 @@
-import random
 
 
 def evaluate(user_submission_file, phase_codename, test_annotation_file=None, **kwargs):
```

Comment:
can you update the order of arguments to be `submission_file, test_annotation_file, phase_codename, **kwargs`

Reply:
@Ram81 I think they will have to change the code in `main.py` to correcly pass the test annotation file if we make it into a positional argument. I made it `None` and placed it add the end so they can pass a default, or skip and use `phase_codename` to find the correct file. Wdyt?

Reply:
I see, that makes sense. In that case can you add a comment in the evaluate method of this form
```
'''
# Load test annotation file for current phase
test_annotation_file = json.loads(open("{phase_codename}_path", "r"))
'''

Reply:
I have made a couple changes accordingly. Please let me know if they look okay.